Journalator_Display/Source/Display/InvestONator/PortfolioDisplay.lua

```
@@ -0,0 +1,934 @@
+    return
+  end
+
+  if not self.AddItemDialog then
+    local dialog = CreateBasicDialog(self, 420, 320)
+
+    dialog.Title:SetText(JOURNALATOR_L_ADD_ITEM or "Add Item")
+
+    -- Amount input (moved above item input)
+    local amountLabel = dialog:CreateFontString(nil, "OVERLAY", "GameFontNormal")
+    amountLabel:SetPoint("TOPLEFT", 20, -60)
+    amountLabel:SetText((JOURNALATOR_L_TARGET_AMOUNT or "Target Amount") .. " (gold)")
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'JOURNALATOR_L_TARGET_AMOUNT'

(W113)